Packs/GSuiteAdmin/Integrations/GSuiteAdmin/README.md

```diff
@@ -1199,3 +1199,193 @@ There is no context output for this command.
 #### Human Readable Output
 
 >User with user key user.test@domain.io deleted successfully.
+
+
+### google-mobiledevice-list
+***
+Retrieves a paginated list that includes company-owned mobile devices.
+
+##### Required Scopes
```

Comment:
Should'nt this be called Permissions? Please check if other integration are using this title.

Reply:
We should discuss it in the demo since no other commands in the README state the desired scopes, the required scopes can be seen in the configuration screen in the product. (Maybe we should also add them to the README?)